app/javascript/components/subject/LastDateExposure.js

```diff
@@ -204,14 +204,41 @@ class LastDateExposure extends React.Component {
             </Row>
             <Row className="pt-2">
               <Col>
-                <Form.Check
-                  size="lg"
-                  label="CONTINUOUS EXPOSURE"
-                  type="switch"
-                  id="continuous_exposure"
-                  checked={this.state.continuous_exposure === true || false}
-                  onChange={() => this.toggleContinuousMonitoringModal()}
-                />
+                {!this.props.patient.monitoring && (
+                  <OverlayTrigger
+                    key="tooltip-ot-ce"
+                    placement="left"
+                    overlay={
+                      <Tooltip id="tooltip-ce">
+                        Continuous Exposure cannot be toggled for records on the Closed line list. If this monitoree requires monitoring due to a Continuous
+                        Exposure, you may update this field after changing Monitoring Status to &quot;Actively Monitoring&quot;
+                      </Tooltip>
+                    }>
+                    <span className="d-inline-block">
+                      <Form.Check
+                        size="lg"
+                        label="CONTINUOUS EXPOSURE"
+                        id="continuous_exposure"
+                        disabled={!this.props.patient.monitoring}
+                        checked={this.state.continuous_exposure === true || false}
+                        onChange={() => this.toggleContinuousMonitoringModal()}
+                      />
+                    </span>
+                  </OverlayTrigger>
+                )}
+                {this.props.patient.monitoring && (
+                  <span className="d-inline-block">
+                    <Form.Check
+                      size="lg"
+                      label="CONTINUOUS EXPOSURE"
+                      id="continuous_exposure"
+                      disabled={!this.props.patient.monitoring}
+                      checked={this.state.continuous_exposure === true || false}
```

Comment:
This can just be `checked={this.state.continuous_exposure}` I think.

Reply:
This is actually done in a bunch of places (and very likely persisted due to copy&pasting from other parts of the source). I am going to look into this and try to figure out why this _may_ have been done in the first place (I believe `if` behaves like an `==` instead of an `===` in js); if I can't find a good reason I will make this change.

Reply:
It would be a slight functionality difference in that `===` will check true equality and without that it will just be checking for `truthiness` - but I think that should be okay. Good idea to see if there was a reason for this in the first place.

Reply:
Made this change